Resfeber/View Controllers/AddTripViewController.swift

```diff
@@ -115,7 +114,7 @@ class AddTripViewController: UIViewController {
                                        startDate: nil,
                                        endDate: nil)
         print("Trip was created: \(trip)")
-        NotificationCenter.default.post(name: NSNotification.Name("load"), object: nil)
+        NotificationCenter.default.post(name: NSNotification.Name.loadData, object: nil)
```

Comment:
the `name` parameter is already of type `NSNotification.Name` so you should only need to pass in `.loadData` here